jetty-quic/quic-server/src/main/java/org/eclipse/jetty/quic/server/QuicServerConnector.java

```diff
@@ -143,6 +145,16 @@ public void setUseOutputDirectByteBuffers(boolean useOutputDirectByteBuffers)
         this.useOutputDirectByteBuffers = useOutputDirectByteBuffers;
     }
 
+    public Path getCertificateWorkPath()
```

Comment:
I'd call this `PemWorkDirectory` because it must be a directory, and it is strictly related to PEM (if Quiche gets in-memory support, we won't need it).

Also, we don't want them in this class, but in `QuicConfiguration`.

Reply:
Done.